### PR TITLE
Payload logging improvement for big files

### PR DESCRIPTION
**Pull request by davidjovanovic**
The response over wss will not be logged from now on until explicitly setting the corresponding log level properties to debug

### CODE REVIEW

File: src/main/java/it/eng/idsa/businesslogic/service/impl/DapsOrbiterServiceImpl.java [hunk 1]
```diff
@@ -118,8 +116,9 @@ public String getJwtToken() {
             logger.error("Error retrieving token:", e);
         } catch (Exception e) {
             logger.error("Something else went wrong:", e);
-        }
-        //settings.setDynamicAttributeToken(dynamicAttributeToken);
+        } finally {
+            jwtResponse.close();
```
Comment (IgorBalog-Eng) [inline src/main/java/it/eng/idsa/businesslogic/service/impl/DapsOrbiterServiceImpl.java:120]: Missing not null check :(

File: src/main/java/it/eng/idsa/businesslogic/service/impl/DapsServiceImpl.java [hunk 2]
```diff
@@ -181,18 +185,16 @@ public boolean verify(String hostname, SSLSession session) {
...
-            if (!responseDaps.isSuccessful())
-                throw new IOException("Unexpected code " + responseDaps);
 
         } catch (IOException |KeyStoreException | NoSuchAlgorithmException | 
         		CertificateException |KeyManagementException | UnrecoverableKeyException e) {
         	logger.error(e);
             return null;
-        	}
+        } finally {
+			responseDaps.close();
```
Comment (IgorBalog-Eng) [inline src/main/java/it/eng/idsa/businesslogic/service/impl/DapsServiceImpl.java:196]: Same like above - not null check

File: src/main/resources/application-RECEIVER.properties [hunk 3]
```diff
@@ -99,16 +99,16 @@ information.model.version=@information.model.version@
 application.basedir=@application.basedir@
 
 # Enable WebSocket over Https -> Disable Idscp to use!
-application.websocket.isEnabled=false
+application.websocket.isEnabled=true
```
Comment (IgorBalog-Eng) [inline src/main/resources/application-RECEIVER.properties:102]: Should we put those "as default" to be false, to use http flow, not wss?

**Review by IgorBalog-Eng — COMMENTED** [review]
Done with code review

File: pom.xml [hunk 4]
```diff
@@ -24,7 +24,7 @@
 		<tomcat.version>9.0.24</tomcat.version>
 		<information.model.version>4.0.0</information.model.version>
 		<application.basedir>${basedir}</application.basedir>
-		<multipart.message.processor.version>1.0.5-SNAPSHOT</multipart.message.processor.version>
+		<multipart.message.processor.version>1.0.6-SNAPSHOT</multipart.message.processor.version>
```
[the remark this answers is not in the document]
Comment (davidjovanovic) [reply pom.xml]: Changed back version of MMP to 1.0.5-SNAPSHOT

File: src/main/java/it/eng/idsa/businesslogic/processor/sender/websocket/client/MessageWebSocketOverHttpSender.java [hunk 5]
```diff
@@ -40,7 +39,7 @@
 
 @Component
 public class MessageWebSocketOverHttpSender {
-    private static final Logger logger = LogManager.getLogger(SenderSendDataToBusinessLogicProcessor.class);
+    private static final Logger logger = LogManager.getLogger(MessageWebSocketOverHttpSender.class);
```
[the remark this answers is not in the document]
Comment (davidjovanovic) [reply src/main/java/it/eng/idsa/businesslogic/processor/sender/websocket/client/MessageWebSocketOverHttpSender.java:42]: Thanks 👍

File: src/main/java/it/eng/idsa/businesslogic/service/impl/DapsOrbiterServiceImpl.java (hunk 1 above)
Comment (davidjovanovic) [reply src/main/java/it/eng/idsa/businesslogic/service/impl/DapsOrbiterServiceImpl.java]: Sorry forgot. Will do ASAP :)

File: src/main/resources/application-RECEIVER.properties (hunk 3 above)
Comment (davidjovanovic) [reply src/main/resources/application-RECEIVER.properties]: Sure, no problem.

**Pull request merged**